bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java

```diff
@@ -127,12 +127,13 @@ public ReplicationWorker(final ZooKeeper zkc,
         this.zkc = zkc;
         this.conf = conf;
         LedgerManagerFactory mFactory = LedgerManagerFactory
-                .newLedgerManagerFactory(this.conf, this.zkc);
+                .newLedgerManagerFactory(
```

Comment:
I would suggest getting layout manager from registration client, rather than construct zk layout manager directly.

Reply:
Thanks, will change it.